README.md

```diff
@@ -87,34 +87,8 @@ pip install forte.spacy
 ```
 
 Let's start by writing a simple processor that analyze POS tags to tokens using the good old NLTK library.
-```python
-import nltk
-
-from forte.processors.base import PackProcessor
-from forte.data.data_pack import DataPack
-from ft.onto.base_ontology import Token
-
-class NLTKPOSTagger(PackProcessor):
-    r"""A wrapper of NLTK pos tagger."""
-
-    def initialize(self, resources, configs):
-        super().initialize(resources, configs)
-        # download the NLTK average perceptron tagger
-        nltk.download("averaged_perceptron_tagger")
-
-    def _process(self, input_pack: DataPack):
-        # get a list of token data entries from `input_pack`
-        # using `DataPack.get()`` method
-        token_entries = input_pack.get(Token)
-        token_texts = [token.text for token in token_entries]
-
-        # use nltk pos tagging module to tag token texts
-        taggings = nltk.pos_tag(token_texts)
-
-        # assign nltk taggings to token attributes
-        for token, tag in zip(token_entries, taggings):
-            token.pos = tag[1]
-```
+<!-- MARKDOWN-AUTO-DOCS:START (CODE:src=tests/forte/readme_test.py=28-52) -->
```

Comment:
btw, if this succeeds eventually, it'd be better if we don't use the line numbers here (because changing the test case itself will render the readme.). I would suggest putting both code snippets in their individual files. And the test cases should be on their own files.

Reply:
is it like two versions of readme code? One for embedding and the other for testing.

Reply:
I also realize we cannot avoid using line numbers. But it would be still better to separate code snippets from the test code (we can actually do unit tests in the test file).

Reply:
- docs
   - readme_code
     - read_me_snippet_1.py
     - ...

- tests
   - readme_code
      - test_read_me_snippet_1.py
      - ...


Or keep all files under `docs/readme_code`

Reply:
The structure looks good.

Btw, this PR passes the CI but we cannot check the doc build. Also, `readthedoc` rendering doesn't work since it starts before the Github action.